src/verification_key.rs

```diff
@@ -72,6 +73,28 @@ impl From<VerificationKeyBytes> for [u8; 32] {
     }
 }
 
+impl<'a> From<SubjectPublicKeyInfo<'a>> for VerificationKeyBytes {
+    fn from(spki: SubjectPublicKeyInfo) -> VerificationKeyBytes {
+        VerificationKeyBytes(spki.subject_public_key.try_into().expect("Ed25519 SubjectPublicKeyInfo doesn't return 32 bytes"))
+    }
+}
+
+#[cfg(feature = "pem")]
+impl From<PublicKeyDocument> for VerificationKeyBytes {
+    fn from(doc: PublicKeyDocument) -> VerificationKeyBytes {
+        let spki = doc.unwrap();
+        VerificationKeyBytes(spki.subject_public_key.try_into().expect("Ed25519 SubjectPublicKeyInfo doesn't return 32 bytes"))
+    }
+}
```

Comment:
I'd probably suggest making these `TryFrom` impls, so if the SPKI is malformed, you can return an error rather than crashing the program.

For PEM specifically, it'd probably be good to add a `FromStr` impl as well